### PR TITLE
docs: add second weekly report template

### PR DESCRIPTION
**Pull request by Yangxiamao**
Fixes: #[Add issue number here]

Changes:

Screenshots of the change:

### CODE REVIEW

File: website/events/2021-08-23-weekly-report.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+本周，有 23 位开发者为 Apache APISIX 提交了 98 个 commit 。感谢本周以下小伙伴为 Apache APISIX 添砖加瓦（排名不分先后），是你们的无私付出，让 Apache APISIX 变得更好！
+
+> juzhiyuan, spacewander, liuxiran, okaybase, bzp2010, shuaijinchao, nic-chen, wmdmomo, KishaniKandasamy, imjoey, Serendipity96, moonming, foolwc, tzssangglass, lingsamuel, tao12345666333, Baoyuantop, yzeng25, Yangxiamao, tokers, jbampton, 1502shivam-singh, RocFang
+
+## Good first issue
+
+
+
+## 本周功能特性亮点
+
```
Comment (tzssangglass) [inline website/events/2021-08-23-weekly-report.md:54]:
```suggestion
- **gzip 插件支持特殊的 * 来匹配任何类型**
  **相关 PR**：https://github.com/apache/apisix/pull/4817
  **贡献者**：[RocFang](https://github.com/RocFang)

- **ext-plugin 插件支持 ExtraInfo**
  **相关 PR**：https://github.com/apache/apisix/pull/4835
  **贡献者**：[spacewander](https://github.com/spacewander)

- **real-ip 插件为 `X-Forwarded-For` 添加特殊处理**
  **相关 PR**：https://github.com/apache/apisix/pull/4852
  **贡献者**：[spacewander](https://github.com/spacewander)

- **uri-blocker 插件支持自定义失败响应**  
  **相关 PR**：https://github.com/apache/apisix/pull/4849
  **贡献者**：[okaybase](https://github.com/okaybase)
```

File: website/events/2021-08-23-weekly-report.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+本周，有 23 位开发者为 Apache APISIX 提交了 98 个 commit 。感谢本周以下小伙伴为 Apache APISIX 添砖加瓦（排名不分先后），是你们的无私付出，让 Apache APISIX 变得更好！
+
+> juzhiyuan, spacewander, liuxiran, okaybase, bzp2010, shuaijinchao, nic-chen, wmdmomo, KishaniKandasamy, imjoey, Serendipity96, moonming, foolwc, tzssangglass, lingsamuel, tao12345666333, Baoyuantop, yzeng25, Yangxiamao, tokers, jbampton, 1502shivam-singh, RocFang
+
+## Good first issue
+
+
+
+## 本周功能特性亮点
```
Comment (tzssangglass) [inline website/events/2021-08-23-weekly-report.md:53]:
```suggestion
## 本周功能特性亮点

```

**Comment by tzssangglass** [comment]
pls fix CI error

File: website/events/2021-08-23-weekly-report.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+---
+title: Apache APISIX 社区周报 ｜ 2021年8月16日～8月22日
+
```
Comment (liuxiran) [inline website/events/2021-08-23-weekly-report.md]: keywords

File: website/events/2021-08-23-weekly-report.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+[Apache APISIX](https://github.com/apache/apisix) 从开源第一天就以社区方式成长，迅速成为全世界最活跃的开源 API 网关项目。这些成就，离不开社区小伙伴们的共同奋斗。
+
+“独行者速，众行者远”。Apache APISIX 社区周报希望可以帮助社区小伙伴们更好地掌握 Apache APISIX 社区的每周进展，方便大家参与到 Apache APISIX 社区中来。
+
+我们还整理了一些适合新来社区的小伙伴们参加的 issue ！感兴趣的同学们，走过路过不要错过！
+
+## 贡献者统计
+
+![second_contributor_static](https://static.apiseven.com/202108/1629726394722-84d86d5a-1b62-4bbd-b681-c42b371c53ec.png)
```
Comment (liuxiran) [inline website/events/2021-08-23-weekly-report.md]:
```suggestion
![weekly_contributor_static_20210823](https://static.apiseven.com/202108/1629726394722-84d86d5a-1b62-4bbd-b681-c42b371c53ec.png)
```

File: website/events/2021-08-23-weekly-report.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+- **uri-blocker 插件支持自定义失败响应**  
+  **相关 PR**：https://github.com/apache/apisix/pull/4849
+  **贡献者**：[okaybase](https://github.com/okaybase)
+
+
+Apache APISIX 的[项目官网](https://apisix.apache.org/)和 Github 上的 [issue](https://github.com/apache/apisix/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) 上已经积累了比较丰富的文档教程和使用经验，如果您遇到问题可以翻阅文档，用关键词在 issue 中搜索，也可以参与 issue 上的讨论，提出自己的想法和实践经验。
+
+## 本周博文推荐
+
```
Comment (liuxiran) [inline website/events/2021-08-23-weekly-report.md:74]: may I ask what is the recommended blogs?

**Review by liuxiran — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

**Pull request merged**